deltachat-ios/View/ProviderInfoCell.swift

```diff
@@ -0,0 +1,111 @@
+        tintColor: UIColor.white
+    )
+
+    private let preparationColorSet: ColorSet = ColorSet(
+        backgroundColor: SystemColor.yellow.uiColor.withAlphaComponent(0.5),
+        tintColor: DcColors.grayTextColor
+    )
+
+    var onInfoButtonPressed: VoidFunction?
+
+    private var hintBackgroundView: UIView = {
+        let view = UIView()
+        return view
+    }()
+
+    private lazy var hintLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 0
+        label.font = UIFont.systemFont(ofSize: fontSize)
+        return label
+    }()
+
+    private lazy var infoButton: UIButton = {
+        let button = UIButton()
+        let title = String.localized("more_info_desktop")
+        button.setTitle(title, for: .normal)
+        button.titleLabel?.font = UIFont.boldSystemFont(ofSize: fontSize)
+        button.addTarget(self, action: #selector(infoButtonPressed(_:)), for: .touchUpInside)
+        return button
+    }()
+
+    init() {
+        super.init(style: .default, reuseIdentifier: nil)
+        setupSubviews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        // ... no highlight
+    }
+
+    private func setupSubviews() {
+
+        let margin: CGFloat = 15
+        let padding: CGFloat = 10
+
+        contentView.addSubview(hintBackgroundView)
+        hintBackgroundView.addSubview(hintLabel)
+        hintBackgroundView.addSubview(infoButton)
+
+        hintBackgroundView.translatesAutoresizingMaskIntoConstraints = false
+        hintLabel.translatesAutoresizingMaskIntoConstraints = false
+        infoButton.translatesAutoresizingMaskIntoConstraints = false
+
+        hintBackgroundView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: margin).isActive = true
+        hintBackgroundView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: margin).isActive = true
+        hintBackgroundView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -margin).isActive = true
+        hintBackgroundView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -margin).isActive = true
+
+        hintLabel.leadingAnchor.constraint(equalTo: hintBackgroundView.leadingAnchor, constant: padding).isActive = true
+        hintLabel.topAnchor.constraint(equalTo: hintBackgroundView.topAnchor, constant: 5).isActive = true
+        hintLabel.trailingAnchor.constraint(equalTo: hintBackgroundView.trailingAnchor, constant: -padding).isActive = true
+
+        infoButton.topAnchor.constraint(equalTo: hintLabel.bottomAnchor, constant: padding).isActive = true
+        infoButton.leadingAnchor.constraint(equalTo: hintBackgroundView.leadingAnchor, constant: padding).isActive = true
+        infoButton.bottomAnchor.constraint(equalTo: hintBackgroundView.bottomAnchor, constant: -5).isActive = true
+    }
```

Comment:
the padding atop of the text should be roughly doubled.

the font has to be a bit larger, maybe sth. in between the current size and the size used for the control labels.